webapp/channels/src/utils/constants.tsx

```diff
@@ -178,12 +178,6 @@ export const TrialPeriodDays = {
 export const suitePluginIds = {
     playbooks: 'playbooks',
 
-    /**
-     * @warning This only applies to the Boards product and will not work with the Boards plugin. Both cases need to
```

Comment:
Could you also remove the `deprecated` warning from the `focalboard` ID below?

Reply:
I thought of that, but decided to leave it as it's technically deprecated still. 0/5 tho

Reply:
I'd lean towards not marking it deprecated since there's legitimate use cases for it still, but 0/5 as well since it's pretty unlikely that we're making any further changes involving it